tinywallet/tinywallet/config.py

```diff
@@ -67,16 +68,15 @@ class CmdArgs:
     def __init__(self):
         self.logLevel = logging.INFO
         self.moduleLevels = {}
-        self.netParams = nets.mainnet
         parser = argparse.ArgumentParser()
         parser.add_argument("--loglevel")
         netGroup = parser.add_mutually_exclusive_group()
         netGroup.add_argument("--simnet", action="store_true", help="use simnet")
         netGroup.add_argument("--testnet", action="store_true", help="use testnet")
         args, unknown = parser.parse_known_args()
         if unknown:
-            exit(f"unknown arguments:{unknown}")
-        self.netParams = None
+            sys.exit(f"unknown arguments:{unknown}")
+        self.netParams = nets.mainnet
```

Comment:
Tests pass if this is set to `nets.mainnet`, and of course it only needs to be set once. :slightly_smiling_face: